### DISCUSSION TITLE
Prisma Seed with Next.js is ran every time I restart `next dev`

### DISCUSSION THREAD

**Discussion by timharek** [prisma/prisma#22555] [Q&A]
### Question

Every time I restart my Next.js development-server with `next dev` my `seed.ts`-file for Prisma is being executed when I interact with the Prisma-client.

Is this expected behaviour?

### How to reproduce (optional)

1. Set up a Next.js project.
2. Follow the official documentation for [seeding with Prisma](https://www.prisma.io/docs/orm/prisma-migrate/workflows/seeding).
3. Interact with your PrismaClient somewhere.
4. See that the `console.log()` is logged in your terminal output. 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["fullTextSearch"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_PRISMA_URL")
}

// Mutliple models
```

```ts
import { PrismaClient } from "@prisma/client";
// Importing other seeding-files for file-splitting

export const prisma = new PrismaClient();

async function main() {
  console.log('Seeding ran')
  const env = process.env.ENV;
  if (env === "dev") {
    // Bunch of different Prisma queries
  }
}

main()
  .then(async () => {
    await prisma.$disconnect();
  })
  .catch(async (e) => {
    // eslint-disable-next-line no-console
    console.error(e);
    await prisma.$disconnect();
    process.exit(1);
  });

```

- OS: macOS 14.1.2
- Database: PostgreSQL
- Node.js version: v20.8.0
- Prisma: v5.7.1
- Next.js: v13.5.6

**Comment by timharek** [comment]
We realized we imported the wrong Primsa client because we were using it in mulitple files. We discovered this when we renamed the PrismaClient.

**Comment by premyedshikar744-ops** [comment]
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["fullTextSearch"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_PRISMA_URL")
}

// Mutliple models
